ArgumentNullException when trying to roll back

**Describe the bug**
When Stryker tries to rollback a mutation, but cannot find the `AssemblyOriginatorKeyFile` for the project, Stryker throws an `ArgumentNullException`.

I cloned and ran Stryker in Debug mode:

In 
```c#
RollbackProcess.Start(CSharpCompilation compiler, ImmutableArray<Diagnostic> diagnostics, bool devMode)`
foreach (var diagnostic in diagnostics.Where(x => x.Severity == DiagnosticSeverity.Error))
{
...
}
```

when the error throws, `diagnostic` has the following values:

```
diagnostic = error CS7027: Error signing output with public key from file 'C:\Users\jonas\source\repos\GenericTests\GenericTests\sgKey.snk' -- Could not find file 'C:\Users\jonas\source\repos\GenericTests\GenericTests\sgKey.snk'.
diagnostic.Location = NoLocation
diagnostic.Location.SourceTree = null
```

**Logs**
From running `dotnet stryker`
```
An error occurred during the mutation test run
System.ArgumentNullException: Value cannot be null.
Parameter name: key
   at System.Collections.Generic.Dictionary`2.FindEntry(TKey key)
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Stryker.Core.Compiling.RollbackProcess.Start(CSharpCompilation compiler, ImmutableArray`1 diagnostics, Boolean devMode) in d:\a\1\s\src\Stryker.Core\Stryker.Core\Compiling\RollbackProcess.cs:line 44
   at Stryker.Core.Compiling.CompilingProcess.TryCompilation(MemoryStream ms, CSharpCompilation compilation, EmitResult previousEmitResult, Boolean devMode, Int32 retryCount) in d:\a\1\s\src\Stryker.Core\Stryker.Core\Compiling\CompilingProcess.cs:line 107
   at Stryker.Core.Compiling.CompilingProcess.Compile(IEnumerable`1 syntaxTrees, MemoryStream ms, Boolean devMode) in d:\a\1\s\src\Stryker.Core\Stryker.Core\Compiling\CompilingProcess.cs:line 75
   at Stryker.Core.MutationTest.MutationTestProcess.Mutate(StrykerOptions options) in d:\a\1\s\src\Stryker.Core\Stryker.Core\MutationTest\MutationTestProcess.cs:line 92
   at Stryker.Core.StrykerRunner.RunMutationTest(StrykerOptions options) in d:\a\1\s\src\Stryker.Core\Stryker.Core\StrykerRunner.cs:line 79
```
[log-20190731.txt](https://github.com/stryker-mutator/stryker-net/files/3453698/log-20190731.txt)

**Expected behavior**
Instead of throwing an `ArgumentNullException` Stryker should stop gracefully.
Optionally support projects that uses signing.

**Desktop:**
 - OS: Windows 10 Version 1903 (OS build 18362.239)
 - Type of project core
 - Framework Version: 
	- library: `netstandard2.0`
	- test project: `netcoreapp2.2`
	- `dotnet --version`: `3.0.100-preview7-012821`
 - Stryker Version beta 0.12.0

**Additional context**

--- a/src/Stryker.Core/Stryker.Core/Initialisation/ProjectInfo.cs
+++ b/src/Stryker.Core/Stryker.Core/Initialisation/ProjectInfo.cs
@@ -102,7 +102,9 @@ namespace Stryker.Core.Initialisation
         private string _assemblyOriginatorKeyFile;
         public string AssemblyOriginatorKeyFile
         {
-            get => _assemblyOriginatorKeyFile ?? _analyzerResult?.Properties?.GetValueOrDefault("AssemblyOriginatorKeyFile").ToFullPath();
+            get => _assemblyOriginatorKeyFile ?? Path.Combine(
+                Path.GetDirectoryName(ProjectFilePath),
+                _analyzerResult?.Properties?.GetValueOrDefault("AssemblyOriginatorKeyFile"));
             set => _assemblyOriginatorKeyFile = value;
         }
 

